add.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import os
+import sys
+import tempfile
+from pathlib import Path
+
+import ssh
+
+
+def run(user, keyfile, userfile=Path("passwd")):
+    fingerprint = ssh.get_fingerprint(Path(keyfile).read_text())
+
+    if ssh.get_github_key(user, fingerprint) is None:
+        print(f"{keyfile} is not registerd for Github user {user}")
+        return 1
+
+    users = userfile.read_text()
+    tmp = Path(".tmp_passwd")
+
+    written = False
+    new_line = f"{user}:{fingerprint}\n"
+    try:
+        with tmp.open("w") as f:
+            for line in users.splitlines():
+                if line.startswith(user + ":"):
+                    f.write(new_line)
+                    written = True
+                else:
+                    f.write(line + "\n")
```

Comment:
hmm, we could rework it to avoid this easily enough.

I think the goal of only 1 active key per user is ok? We could allow multiple, if people are working from different machines perhaps

Reply:
yeah 1 key is fine